mods/d2k/weapons.yaml

```diff
@@ -717,8 +717,15 @@ Shrapnel:
 		ImpactSound: EXPLLG5.WAV
 
 SpiceExplosion:
+	Report: EXPLMD1.WAV
+	Projectile: Bullet
+		Speed: 50, 75
+		High: true
+		Angle: 91, 264
+		Trail: large_trail
+		Image: large_trail
```

Comment:
This may or may not want to be changed to the "null" sprite. I think It looks a little bit better like this, however.